Install fails on Ubuntu 22.04.1 LTS (and 20.04.5)

G'day team, 
I have been trying to install charts-plugin from the Appstore but I get a failed result.
SignalK is installed and working on a 22.04 ubuntu system. Freeboard works fine with online maps, but I wan to use offline maps and need to install the charts-plugins. (unless there's another way to get offline maps on freeboard without the plugin)
According to Openplotter documentation Ubuntu 22.04 should be supported. (I previously tried on 20.04.5 with same results)
I have installed Openplotter and SignalK following OP docs without issues.
I have installed other plugins and they all seem to work except this one. 
Any help or suggestions will be appreciated
This is the log:

```
0 verbose cli /usr/bin/node /usr/bin/npm
1 info using npm@8.19.2
2 info using node@v16.17.1
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 3ms
5 timing config:load:file:/usr/lib/node_modules/npm/npmrc Completed in 2ms
6 timing config:load:builtin Completed in 2ms
7 timing config:load:cli Completed in 7ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/home/raetara/.signalk/.npmrc Completed in 2ms
10 timing config:load:project Completed in 6ms
11 timing config:load:file:/home/raetara/.npmrc Completed in 0ms
12 timing config:load:user Completed in 1ms
13 timing config:load:file:/usr/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:validate Completed in 1ms
16 timing config:load:credentials Completed in 1ms
17 timing config:load:setEnvs Completed in 1ms
18 timing config:load Completed in 23ms
19 timing npm:load:configload Completed in 23ms
20 timing npm:load:mkdirpcache Completed in 9ms
21 timing npm:load:mkdirplogs Completed in 1ms
22 verbose title npm install @signalk/charts-plugin
23 verbose argv "install" "--save" "@signalk/charts-plugin"
24 timing npm:load:setTitle Completed in 2ms
25 timing config:load:flatten Completed in 5ms
26 timing npm:load:display Completed in 8ms
27 verbose logfile logs-max:10 dir:/home/raetara/.npm/_logs
28 verbose logfile /home/raetara/.npm/_logs/2022-09-28T03_06_57_754Z-debug-0.log
29 timing npm:load:logFile Completed in 8ms
30 timing npm:load:timers Completed in 1ms
31 timing npm:load:configScope Completed in 0ms
32 timing npm:load Completed in 53ms
33 timing arborist:ctor Completed in 1ms
34 silly logfile done cleaning log files
35 timing arborist:ctor Completed in 0ms
36 verbose shrinkwrap failed to load node_modules/.package-lock.json out of date, updated: node_modules
37 verbose shrinkwrap failed to load node_modules/.package-lock.json out of date, updated: node_modules
38 timing idealTree:init Completed in 357ms
39 timing idealTree:userRequests Completed in 2ms
40 silly idealTree buildDeps
41 silly fetch manifest @signalk/charts-plugin@*
42 http fetch GET 200 https://registry.npmjs.org/@signalk%2fcharts-plugin 1175ms (cache revalidated)
43 silly placeDep ROOT @signalk/charts-plugin@2.3.0 OK for: signalk-server-config@0.0.1 want: *
44 silly fetch manifest @mapbox/mbtiles@0.10.0
45 silly fetch manifest baconjs@1.0.1
46 silly fetch manifest bluebird@3.5.1
47 silly fetch manifest debug@3.1.0
48 silly fetch manifest lodash@^4.17.11
49 silly fetch manifest xml2js@0.4.19
50 http fetch GET 200 https://registry.npmjs.org/xml2js 140ms (cache revalidated)
51 http fetch GET 200 https://registry.npmjs.org/debug 188ms (cache revalidated)
52 http fetch GET 200 https://registry.npmjs.org/bluebird 194ms (cache revalidated)
53 http fetch GET 200 https://registry.npmjs.org/baconjs 331ms (cache revalidated)
54 http fetch GET 200 https://registry.npmjs.org/lodash 335ms (cache revalidated)
55 http fetch GET 200 https://registry.npmjs.org/@mapbox%2fmbtiles 844ms (cache revalidated)
56 timing idealTree:#root Completed in 2048ms
57 silly placeDep ROOT @mapbox/mbtiles@0.10.0 OK for: @signalk/charts-plugin@2.3.0 want: 0.10.0
58 silly placeDep ROOT baconjs@1.0.1 OK for: @signalk/charts-plugin@2.3.0 want: 1.0.1
59 silly placeDep ROOT bluebird@3.5.1 OK for: @signalk/charts-plugin@2.3.0 want: 3.5.1
60 silly placeDep ROOT debug@3.1.0 OK for: @signalk/charts-plugin@2.3.0 want: 3.1.0
61 silly placeDep ROOT lodash@4.17.21 OK for: @signalk/charts-plugin@2.3.0 want: ^4.17.11
62 silly placeDep ROOT xml2js@0.4.19 OK for: @signalk/charts-plugin@2.3.0 want: 0.4.19
63 silly fetch manifest d3-queue@~2.0.3
64 silly fetch manifest @mapbox/tiletype@0.3.x
65 silly fetch manifest sqlite3@4.x
66 silly fetch manifest @mapbox/sphericalmercator@~1.1.0
67 silly fetch manifest ms@2.0.0
68 silly fetch manifest sax@>=0.6.0
69 silly fetch manifest xmlbuilder@~9.0.1
70 http fetch GET 200 https://registry.npmjs.org/sax 95ms (cache revalidated)
71 http fetch GET 200 https://registry.npmjs.org/sqlite3 115ms (cache revalidated)
72 http fetch GET 200 https://registry.npmjs.org/ms 163ms (cache revalidated)
73 http fetch GET 200 https://registry.npmjs.org/d3-queue 387ms (cache revalidated)
74 http fetch GET 200 https://registry.npmjs.org/xmlbuilder 380ms (cache revalidated)
75 http fetch GET 200 https://registry.npmjs.org/@mapbox%2fsphericalmercator 1386ms (cache revalidated)
76 http fetch GET 200 https://registry.npmjs.org/@mapbox%2ftiletype 2029ms (cache revalidated)
77 timing idealTree:node_modules/@signalk/charts-plugin Completed in 2054ms
78 silly placeDep ROOT @mapbox/sphericalmercator@1.1.0 OK for: @mapbox/mbtiles@0.10.0 want: ~1.1.0
79 silly placeDep ROOT @mapbox/tiletype@0.3.1 OK for: @mapbox/mbtiles@0.10.0 want: 0.3.x
80 silly placeDep ROOT d3-queue@2.0.3 OK for: @mapbox/mbtiles@0.10.0 want: ~2.0.3
81 silly placeDep ROOT sqlite3@4.2.0 OK for: @mapbox/mbtiles@0.10.0 want: 4.x
82 silly fetch manifest nan@^2.12.1
83 silly fetch manifest node-pre-gyp@^0.11.0
84 http fetch GET 200 https://registry.npmjs.org/nan 49ms (cache revalidated)
85 http fetch GET 200 https://registry.npmjs.org/node-pre-gyp 60ms (cache revalidated)
86 timing idealTree:node_modules/@mapbox/mbtiles Completed in 76ms
87 timing idealTree:node_modules/@mapbox/sphericalmercator Completed in 0ms
88 timing idealTree:node_modules/@mapbox/tiletype Completed in 1ms
89 timing idealTree:node_modules/baconjs Completed in 0ms
90 timing idealTree:node_modules/bluebird Completed in 0ms
91 timing idealTree:node_modules/d3-queue Completed in 0ms
92 silly placeDep ROOT ms@2.0.0 OK for: debug@3.1.0 want: 2.0.0
93 timing idealTree:node_modules/debug Completed in 2ms
94 timing idealTree:node_modules/lodash Completed in 0ms
95 timing idealTree:node_modules/ms Completed in 1ms
96 silly placeDep ROOT nan@2.16.0 OK for: sqlite3@4.2.0 want: ^2.12.1
97 silly placeDep ROOT node-pre-gyp@0.11.0 OK for: sqlite3@4.2.0 want: ^0.11.0
98 silly fetch manifest detect-libc@^1.0.2
99 silly fetch manifest needle@^2.2.1
100 silly fetch manifest nopt@^4.0.1
101 silly fetch manifest npm-packlist@^1.1.6
102 silly fetch manifest npmlog@^4.0.2
103 silly fetch manifest rc@^1.2.7
104 silly fetch manifest rimraf@^2.6.1
105 silly fetch manifest semver@^5.3.0
106 silly fetch manifest tar@^4
107 http fetch GET 200 https://registry.npmjs.org/detect-libc 67ms (cache revalidated)
108 http fetch GET 200 https://registry.npmjs.org/nopt 74ms (cache revalidated)
109 http fetch GET 200 https://registry.npmjs.org/npm-packlist 81ms (cache revalidated)
110 http fetch GET 200 https://registry.npmjs.org/tar 88ms (cache revalidated)
111 http fetch GET 200 https://registry.npmjs.org/rimraf 102ms (cache revalidated)
112 http fetch GET 200 https://registry.npmjs.org/rc 108ms (cache revalidated)
113 http fetch GET 200 https://registry.npmjs.org/semver 158ms (cache revalidated)
114 http fetch GET 200 https://registry.npmjs.org/npmlog 166ms (cache revalidated)
115 http fetch GET 200 https://registry.npmjs.org/needle 977ms (cache revalidated)
116 timing idealTree:node_modules/sqlite3 Completed in 998ms
117 timing idealTree:node_modules/nan Completed in 0ms
118 silly placeDep ROOT detect-libc@1.0.3 OK for: node-pre-gyp@0.11.0 want: ^1.0.2
119 silly placeDep ROOT needle@2.9.1 OK for: node-pre-gyp@0.11.0 want: ^2.2.1
120 silly placeDep ROOT nopt@4.0.3 OK for: node-pre-gyp@0.11.0 want: ^4.0.1
121 silly placeDep ROOT npm-packlist@1.4.8 OK for: node-pre-gyp@0.11.0 want: ^1.1.6
122 silly placeDep ROOT npmlog@4.1.2 OK for: node-pre-gyp@0.11.0 want: ^4.0.2
123 silly placeDep ROOT rc@1.2.8 OK for: node-pre-gyp@0.11.0 want: ^1.2.7
124 silly placeDep ROOT rimraf@2.7.1 OK for: node-pre-gyp@0.11.0 want: ^2.6.1
125 silly placeDep ROOT semver@5.7.1 OK for: node-pre-gyp@0.11.0 want: ^5.3.0
126 silly placeDep ROOT tar@4.4.19 OK for: node-pre-gyp@0.11.0 want: ^4
127 silly fetch manifest debug@^3.2.6
128 silly fetch manifest iconv-lite@^0.4.4
129 silly fetch manifest sax@^1.2.4
130 silly fetch manifest abbrev@1
131 silly fetch manifest osenv@^0.1.4
132 silly fetch manifest ignore-walk@^3.0.1
133 silly fetch manifest npm-bundled@^1.0.1
134 silly fetch manifest npm-normalize-package-bin@^1.0.1
135 silly fetch manifest are-we-there-yet@~1.1.2
136 silly fetch manifest console-control-strings@~1.1.0
137 silly fetch manifest gauge@~2.7.3
138 silly fetch manifest set-blocking@~2.0.0
139 silly fetch manifest deep-extend@^0.6.0
140 silly fetch manifest ini@~1.3.0
141 silly fetch manifest strip-json-comments@~2.0.1
142 silly fetch manifest glob@^7.1.3
143 silly fetch manifest chownr@^1.1.4
144 silly fetch manifest fs-minipass@^1.2.7
145 silly fetch manifest minipass@^2.9.0
146 silly fetch manifest minizlib@^1.3.3
147 silly fetch manifest yallist@^3.1.1
148 http fetch GET 200 https://registry.npmjs.org/console-control-strings 185ms (cache revalidated)
149 http fetch GET 200 https://registry.npmjs.org/osenv 195ms (cache revalidated)
150 http fetch GET 200 https://registry.npmjs.org/npm-bundled 193ms (cache revalidated)
151 http fetch GET 200 https://registry.npmjs.org/iconv-lite 212ms (cache revalidated)
152 http fetch GET 200 https://registry.npmjs.org/ini 200ms (cache revalidated)
153 http fetch GET 200 https://registry.npmjs.org/gauge 208ms (cache revalidated)
154 http fetch GET 200 https://registry.npmjs.org/set-blocking 216ms (cache revalidated)
155 http fetch GET 200 https://registry.npmjs.org/chownr 215ms (cache revalidated)
156 http fetch GET 200 https://registry.npmjs.org/are-we-there-yet 229ms (cache revalidated)
157 http fetch GET 200 https://registry.npmjs.org/minipass 214ms (cache revalidated)
158 http fetch GET 200 https://registry.npmjs.org/minizlib 215ms (cache revalidated)
159 http fetch GET 200 https://registry.npmjs.org/deep-extend 234ms (cache revalidated)
160 http fetch GET 200 https://registry.npmjs.org/npm-normalize-package-bin 246ms (cache revalidated)
161 http fetch GET 200 https://registry.npmjs.org/fs-minipass 229ms (cache revalidated)
162 http fetch GET 200 https://registry.npmjs.org/yallist 237ms (cache revalidated)
163 http fetch GET 200 https://registry.npmjs.org/strip-json-comments 251ms (cache revalidated)
164 http fetch GET 200 https://registry.npmjs.org/ignore-walk 406ms (cache revalidated)
165 http fetch GET 200 https://registry.npmjs.org/glob 461ms (cache revalidated)
166 http fetch GET 200 https://registry.npmjs.org/abbrev 1025ms (cache revalidated)
167 timing idealTree:node_modules/node-pre-gyp Completed in 1064ms
168 timing idealTree:node_modules/detect-libc Completed in 0ms
169 silly placeDep node_modules/needle debug@3.2.7 OK for: needle@2.9.1 want: ^3.2.6
170 silly placeDep ROOT iconv-lite@0.4.24 OK for: needle@2.9.1 want: ^0.4.4
171 silly placeDep ROOT sax@1.2.4 OK for: needle@2.9.1 want: ^1.2.4
172 silly fetch manifest ms@^2.1.1
173 silly fetch manifest safer-buffer@>= 2.1.2 < 3
174 http fetch GET 200 https://registry.npmjs.org/safer-buffer 53ms (cache revalidated)
175 timing idealTree:node_modules/needle Completed in 63ms
176 silly placeDep ROOT safer-buffer@2.1.2 OK for: iconv-lite@0.4.24 want: >= 2.1.2 < 3
177 timing idealTree:node_modules/iconv-lite Completed in 2ms
178 silly placeDep ROOT abbrev@1.1.1 OK for: nopt@4.0.3 want: 1
179 silly placeDep ROOT osenv@0.1.5 OK for: nopt@4.0.3 want: ^0.1.4
180 silly fetch manifest os-homedir@^1.0.0
181 silly fetch manifest os-tmpdir@^1.0.0
182 http fetch GET 200 https://registry.npmjs.org/os-tmpdir 60ms (cache revalidated)
183 http fetch GET 200 https://registry.npmjs.org/os-homedir 63ms (cache revalidated)
184 timing idealTree:node_modules/nopt Completed in 67ms
185 timing idealTree:node_modules/abbrev Completed in 0ms
186 silly placeDep ROOT ignore-walk@3.0.4 OK for: npm-packlist@1.4.8 want: ^3.0.1
187 silly placeDep ROOT npm-bundled@1.1.2 OK for: npm-packlist@1.4.8 want: ^1.0.1
188 silly placeDep ROOT npm-normalize-package-bin@1.0.1 OK for: npm-packlist@1.4.8 want: ^1.0.1
189 silly fetch manifest minimatch@^3.0.4
190 http fetch GET 200 https://registry.npmjs.org/minimatch 50ms (cache revalidated)
191 timing idealTree:node_modules/npm-packlist Completed in 59ms
192 silly placeDep ROOT minimatch@3.1.2 OK for: ignore-walk@3.0.4 want: ^3.0.4
193 silly fetch manifest brace-expansion@^1.1.7
194 http fetch GET 200 https://registry.npmjs.org/brace-expansion 50ms (cache revalidated)
195 timing idealTree:node_modules/ignore-walk Completed in 56ms
196 silly placeDep ROOT brace-expansion@1.1.11 OK for: minimatch@3.1.2 want: ^1.1.7
197 silly fetch manifest balanced-match@^1.0.0
198 silly fetch manifest concat-map@0.0.1
199 http fetch GET 200 https://registry.npmjs.org/balanced-match 47ms (cache revalidated)
200 http fetch GET 200 https://registry.npmjs.org/concat-map 52ms (cache revalidated)
201 timing idealTree:node_modules/minimatch Completed in 58ms
202 silly placeDep ROOT balanced-match@1.0.2 OK for: brace-expansion@1.1.11 want: ^1.0.0
203 silly placeDep ROOT concat-map@0.0.1 OK for: brace-expansion@1.1.11 want: 0.0.1
204 timing idealTree:node_modules/brace-expansion Completed in 6ms
205 timing idealTree:node_modules/balanced-match Completed in 0ms
206 timing idealTree:node_modules/concat-map Completed in 0ms
207 timing idealTree:node_modules/npm-bundled Completed in 0ms
208 timing idealTree:node_modules/npm-normalize-package-bin Completed in 0ms
209 silly placeDep ROOT are-we-there-yet@1.1.7 OK for: npmlog@4.1.2 want: ~1.1.2
210 silly placeDep ROOT console-control-strings@1.1.0 OK for: npmlog@4.1.2 want: ~1.1.0
211 silly placeDep ROOT gauge@2.7.4 OK for: npmlog@4.1.2 want: ~2.7.3
212 silly placeDep ROOT set-blocking@2.0.0 OK for: npmlog@4.1.2 want: ~2.0.0
213 silly fetch manifest delegates@^1.0.0
214 silly fetch manifest readable-stream@^2.0.6
215 silly fetch manifest aproba@^1.0.3
216 silly fetch manifest has-unicode@^2.0.0
217 silly fetch manifest object-assign@^4.1.0
218 silly fetch manifest signal-exit@^3.0.0
219 silly fetch manifest string-width@^1.0.1
220 silly fetch manifest strip-ansi@^3.0.1
221 silly fetch manifest wide-align@^1.1.0
222 http fetch GET 200 https://registry.npmjs.org/readable-stream 93ms (cache revalidated)
223 http fetch GET 200 https://registry.npmjs.org/object-assign 95ms (cache revalidated)
224 http fetch GET 200 https://registry.npmjs.org/signal-exit 99ms (cache revalidated)
225 http fetch GET 200 https://registry.npmjs.org/has-unicode 104ms (cache revalidated)
226 http fetch GET 200 https://registry.npmjs.org/strip-ansi 100ms (cache revalidated)
227 http fetch GET 200 https://registry.npmjs.org/string-width 103ms (cache revalidated)
228 http fetch GET 200 https://registry.npmjs.org/aproba 109ms (cache revalidated)
229 http fetch GET 200 https://registry.npmjs.org/wide-align 102ms (cache revalidated)
230 http fetch GET 200 https://registry.npmjs.org/delegates 115ms (cache revalidated)
231 timing idealTree:node_modules/npmlog Completed in 128ms
232 silly placeDep ROOT delegates@1.0.0 OK for: are-we-there-yet@1.1.7 want: ^1.0.0
233 silly placeDep node_modules/are-we-there-yet readable-stream@2.3.7 OK for: are-we-there-yet@1.1.7 want: ^2.0.6
234 silly fetch manifest isarray@~1.0.0
235 silly fetch manifest process-nextick-args@~2.0.0
236 silly fetch manifest safe-buffer@~5.1.1
237 silly fetch manifest string_decoder@~1.1.1
238 http fetch GET 200 https://registry.npmjs.org/process-nextick-args 63ms (cache revalidated)
239 http fetch GET 200 https://registry.npmjs.org/isarray 68ms (cache revalidated)
240 http fetch GET 200 https://registry.npmjs.org/string_decoder 70ms (cache revalidated)
241 http fetch GET 200 https://registry.npmjs.org/safe-buffer 88ms (cache revalidated)
242 timing idealTree:node_modules/are-we-there-yet Completed in 104ms
243 timing idealTree:node_modules/console-control-strings Completed in 0ms
244 timing idealTree:node_modules/delegates Completed in 0ms
245 silly placeDep ROOT aproba@1.2.0 OK for: gauge@2.7.4 want: ^1.0.3
246 silly placeDep ROOT has-unicode@2.0.1 OK for: gauge@2.7.4 want: ^2.0.0
247 silly placeDep ROOT object-assign@4.1.1 OK for: gauge@2.7.4 want: ^4.1.0
248 silly placeDep ROOT signal-exit@3.0.7 OK for: gauge@2.7.4 want: ^3.0.0
249 silly placeDep ROOT string-width@1.0.2 OK for: gauge@2.7.4 want: ^1.0.1
250 silly placeDep ROOT strip-ansi@3.0.1 OK for: gauge@2.7.4 want: ^3.0.1
251 silly placeDep ROOT wide-align@1.1.5 OK for: gauge@2.7.4 want: ^1.1.0
252 silly fetch manifest code-point-at@^1.0.0
253 silly fetch manifest is-fullwidth-code-point@^1.0.0
254 silly fetch manifest ansi-regex@^2.0.0
255 http fetch GET 200 https://registry.npmjs.org/is-fullwidth-code-point 62ms (cache revalidated)
256 http fetch GET 200 https://registry.npmjs.org/ansi-regex 70ms (cache revalidated)
257 http fetch GET 200 https://registry.npmjs.org/code-point-at 73ms (cache revalidated)
258 timing idealTree:node_modules/gauge Completed in 87ms
259 timing idealTree:node_modules/aproba Completed in 0ms
260 timing idealTree:node_modules/has-unicode Completed in 0ms
261 timing idealTree:node_modules/object-assign Completed in 0ms
262 silly placeDep ROOT os-homedir@1.0.2 OK for: osenv@0.1.5 want: ^1.0.0
263 silly placeDep ROOT os-tmpdir@1.0.2 OK for: osenv@0.1.5 want: ^1.0.0
264 timing idealTree:node_modules/osenv Completed in 4ms
265 timing idealTree:node_modules/os-homedir Completed in 0ms
266 timing idealTree:node_modules/os-tmpdir Completed in 0ms
267 silly placeDep ROOT deep-extend@0.6.0 OK for: rc@1.2.8 want: ^0.6.0
268 silly placeDep ROOT ini@1.3.8 OK for: rc@1.2.8 want: ~1.3.0
269 silly placeDep ROOT strip-json-comments@2.0.1 OK for: rc@1.2.8 want: ~2.0.1
270 timing idealTree:node_modules/rc Completed in 5ms
271 timing idealTree:node_modules/deep-extend Completed in 0ms
272 timing idealTree:node_modules/ini Completed in 0ms
273 silly placeDep ROOT glob@7.2.3 OK for: rimraf@2.7.1 want: ^7.1.3
274 silly fetch manifest fs.realpath@^1.0.0
275 silly fetch manifest inflight@^1.0.4
276 silly fetch manifest once@^1.3.0
277 silly fetch manifest path-is-absolute@^1.0.0
278 http fetch GET 200 https://registry.npmjs.org/inflight 89ms (cache revalidated)
279 http fetch GET 200 https://registry.npmjs.org/path-is-absolute 88ms (cache revalidated)
280 http fetch GET 200 https://registry.npmjs.org/once 91ms (cache revalidated)
281 http fetch GET 200 https://registry.npmjs.org/fs.realpath 95ms (cache revalidated)
282 timing idealTree:node_modules/rimraf Completed in 100ms
283 silly placeDep ROOT fs.realpath@1.0.0 OK for: glob@7.2.3 want: ^1.0.0
284 silly placeDep ROOT inflight@1.0.6 OK for: glob@7.2.3 want: ^1.0.4
285 silly placeDep ROOT once@1.4.0 OK for: glob@7.2.3 want: ^1.3.0
286 silly placeDep ROOT path-is-absolute@1.0.1 OK for: glob@7.2.3 want: ^1.0.0
287 silly fetch manifest wrappy@1
288 http fetch GET 200 https://registry.npmjs.org/wrappy 45ms (cache revalidated)
289 timing idealTree:node_modules/glob Completed in 53ms
290 timing idealTree:node_modules/fs.realpath Completed in 0ms
291 silly placeDep ROOT wrappy@1.0.2 OK for: inflight@1.0.6 want: 1
292 timing idealTree:node_modules/inflight Completed in 2ms
293 timing idealTree:node_modules/once Completed in 0ms
294 timing idealTree:node_modules/path-is-absolute Completed in 0ms
295 timing idealTree:node_modules/safer-buffer Completed in 0ms
296 timing idealTree:node_modules/sax Completed in 0ms
297 timing idealTree:node_modules/semver Completed in 0ms
298 timing idealTree:node_modules/set-blocking Completed in 0ms
299 timing idealTree:node_modules/signal-exit Completed in 0ms
300 silly placeDep ROOT code-point-at@1.1.0 OK for: string-width@1.0.2 want: ^1.0.0
301 silly placeDep ROOT is-fullwidth-code-point@1.0.0 OK for: string-width@1.0.2 want: ^1.0.0
302 silly fetch manifest number-is-nan@^1.0.0
303 http fetch GET 200 https://registry.npmjs.org/number-is-nan 47ms (cache revalidated)
304 timing idealTree:node_modules/string-width Completed in 52ms
305 timing idealTree:node_modules/code-point-at Completed in 0ms
306 silly placeDep ROOT number-is-nan@1.0.1 OK for: is-fullwidth-code-point@1.0.0 want: ^1.0.0
307 timing idealTree:node_modules/is-fullwidth-code-point Completed in 2ms
308 timing idealTree:node_modules/number-is-nan Completed in 0ms
309 silly placeDep ROOT ansi-regex@2.1.1 OK for: strip-ansi@3.0.1 want: ^2.0.0
310 timing idealTree:node_modules/strip-ansi Completed in 2ms
311 timing idealTree:node_modules/ansi-regex Completed in 0ms
312 timing idealTree:node_modules/strip-json-comments Completed in 0ms
313 silly placeDep ROOT chownr@1.1.4 OK for: tar@4.4.19 want: ^1.1.4
314 silly placeDep ROOT fs-minipass@1.2.7 OK for: tar@4.4.19 want: ^1.2.7
315 silly placeDep ROOT minipass@2.9.0 OK for: tar@4.4.19 want: ^2.9.0
316 silly placeDep ROOT minizlib@1.3.3 OK for: tar@4.4.19 want: ^1.3.3
317 silly placeDep ROOT yallist@3.1.1 OK for: tar@4.4.19 want: ^3.1.1
318 timing idealTree:node_modules/tar Completed in 10ms
319 timing idealTree:node_modules/chownr Completed in 0ms
320 timing idealTree:node_modules/fs-minipass Completed in 0ms
321 timing idealTree:node_modules/minipass Completed in 0ms
322 timing idealTree:node_modules/minizlib Completed in 0ms
323 timing idealTree:node_modules/wide-align Completed in 0ms
324 timing idealTree:node_modules/wrappy Completed in 0ms
325 silly placeDep ROOT xmlbuilder@9.0.7 OK for: xml2js@0.4.19 want: ~9.0.1
326 timing idealTree:node_modules/xml2js Completed in 2ms
327 timing idealTree:node_modules/xmlbuilder Completed in 0ms
328 timing idealTree:node_modules/yallist Completed in 0ms
329 silly placeDep node_modules/are-we-there-yet isarray@1.0.0 OK for: readable-stream@2.3.7 want: ~1.0.0
330 silly placeDep ROOT process-nextick-args@2.0.1 OK for: readable-stream@2.3.7 want: ~2.0.0
331 silly placeDep node_modules/are-we-there-yet safe-buffer@5.1.2 OK for: readable-stream@2.3.7 want: ~5.1.1
332 silly placeDep node_modules/are-we-there-yet string_decoder@1.1.1 OK for: readable-stream@2.3.7 want: ~1.1.1
333 timing idealTree:node_modules/are-we-there-yet/node_modules/readable-stream Completed in 11ms
334 timing idealTree:node_modules/process-nextick-args Completed in 0ms
335 timing idealTree:node_modules/are-we-there-yet/node_modules/isarray Completed in 0ms
336 timing idealTree:node_modules/are-we-there-yet/node_modules/safe-buffer Completed in 0ms
337 timing idealTree:node_modules/are-we-there-yet/node_modules/string_decoder Completed in 0ms
338 silly placeDep node_modules/needle ms@2.1.3 OK for: debug@3.2.7 want: ^2.1.1
339 timing idealTree:node_modules/needle/node_modules/debug Completed in 3ms
340 timing idealTree:node_modules/needle/node_modules/ms Completed in 0ms
341 timing idealTree:buildDeps Completed in 7129ms
342 timing idealTree:fixDepFlags Completed in 3ms
343 timing idealTree Completed in 7496ms
344 timing reify:loadTrees Completed in 7498ms
345 timing reify:diffTrees Completed in 10ms
346 silly reify moves {}
347 timing reify:retireShallow Completed in 2ms
348 timing reify:createSparse Completed in 23ms
349 timing reify:loadBundles Completed in 0ms
350 silly audit bulk request {
350 silly audit   'abort-controller': [ '3.0.0' ],
350 silly audit   'abstract-leveldown': [ '6.2.3' ],
350 silly audit   argsarray: [ '0.0.1' ],
350 silly audit   'base64-js': [ '1.5.1' ],
350 silly audit   buffer: [ '5.7.1' ],
350 silly audit   'clone-buffer': [ '1.0.0' ],
350 silly audit   'core-util-is': [ '1.0.3' ],
350 silly audit   'buffer-from': [ '1.1.2' ],
350 silly audit   'double-ended-queue': [ '2.1.0-0' ],
350 silly audit   'deferred-leveldown': [ '5.3.0' ],
350 silly audit   'encoding-down': [ '6.3.0' ],
350 silly audit   'end-stream': [ '0.1.0' ],
350 silly audit   errno: [ '0.1.8' ],
350 silly audit   'event-target-shim': [ '5.0.1' ],
350 silly audit   'fetch-cookie': [ '0.11.0' ],
350 silly audit   'freeboard-sk-helper': [ '1.5.0' ],
350 silly audit   'geojson-validation': [ '0.2.1' ],
350 silly audit   immediate: [ '3.3.0' ],
350 silly audit   ieee754: [ '1.2.1' ],
350 silly audit   inherits: [ '2.0.4' ],
350 silly audit   isarray: [ '0.0.1', '1.0.0' ],
350 silly audit   level: [ '6.0.1' ],
350 silly audit   'level-codec': [ '9.0.2' ],
350 silly audit   'level-concat-iterator': [ '2.0.1' ],
350 silly audit   'level-errors': [ '2.0.1' ],
350 silly audit   'level-iterator-stream': [ '4.0.2' ],
350 silly audit   'readable-stream': [ '3.6.0', '1.1.14', '0.0.4', '2.3.7' ],
350 silly audit   string_decoder: [ '1.3.0', '0.10.31', '1.1.1' ],
350 silly audit   'level-js': [ '5.0.2' ],
350 silly audit   'level-packager': [ '5.1.1' ],
350 silly audit   'level-supports': [ '1.0.1' ],
350 silly audit   'level-write-stream': [ '1.0.0' ],
350 silly audit   leveldown: [ '5.6.0' ],
350 silly audit   levelup: [ '4.4.0' ],
350 silly audit   ltgt: [ '2.2.1' ],
350 silly audit   minimist: [ '1.2.6' ],
350 silly audit   mkdirp: [ '0.5.6' ],
350 silly audit   'napi-macros': [ '2.0.0' ],
350 silly audit   'node-fetch': [ '2.6.7' ],
350 silly audit   'node-gyp-build': [ '4.1.1' ],
350 silly audit   pouchdb: [ '7.3.0' ],
350 silly audit   uuid: [ '8.3.2', '3.4.0' ],
350 silly audit   'pouchdb-abstract-mapreduce': [ '7.3.0' ],
350 silly audit   'pouchdb-binary-utils': [ '7.3.0' ],
350 silly audit   'pouchdb-collate': [ '7.3.0' ],
350 silly audit   'pouchdb-collections': [ '7.3.0' ],
350 silly audit   'pouchdb-errors': [ '7.3.0' ],
350 silly audit   'pouchdb-fetch': [ '7.3.0' ],
350 silly audit   'pouchdb-find': [ '7.3.0' ],
350 silly audit   'pouchdb-mapreduce-utils': [ '7.3.0' ],
350 silly audit   'pouchdb-md5': [ '7.3.0' ],
350 silly audit   'pouchdb-selector-core': [ '7.3.0' ],
350 silly audit   'pouchdb-utils': [ '7.3.0' ],
350 silly audit   prr: [ '1.0.1' ],
350 silly audit   psl: [ '1.9.0' ],
350 silly audit   punycode: [ '2.1.1' ],
350 silly audit   querystringify: [ '2.2.0' ],
350 silly audit   'requires-port': [ '1.0.0' ],
350 silly audit   rxjs: [ '7.5.7' ],
350 silly audit   'safe-buffer': [ '5.2.1', '5.1.2' ],
350 silly audit   'sk-resources-fs': [ '1.3.0' ],
350 silly audit   'spark-md5': [ '3.0.2' ],
350 silly audit   through2: [ '3.0.2' ],
350 silly audit   'tough-cookie': [ '4.1.2' ],
350 silly audit   tr46: [ '0.0.3' ],
350 silly audit   tslib: [ '2.4.0' ],
350 silly audit   universalify: [ '0.2.0' ],
350 silly audit   'url-parse': [ '1.5.10' ],
350 silly audit   'util-deprecate': [ '1.0.2' ],
350 silly audit   vuvuzela: [ '1.0.3' ],
350 silly audit   'webidl-conversions': [ '3.0.1' ],
350 silly audit   'whatwg-url': [ '5.0.0' ],
350 silly audit   'write-stream': [ '0.4.3' ],
350 silly audit   xtend: [ '4.0.2' ],
350 silly audit   '@panaaj/sk-types': [ '1.1.1' ],
350 silly audit   '@signalk/course-provider': [ '1.0.0-beta.4a' ],
350 silly audit   '@signalk/charts-plugin': [ '2.3.0' ],
350 silly audit   '@mapbox/mbtiles': [ '0.10.0' ],
350 silly audit   baconjs: [ '1.0.1' ],
350 silly audit   bluebird: [ '3.5.1' ],
350 silly audit   debug: [ '3.1.0', '3.2.7' ],
350 silly audit   lodash: [ '4.17.21' ],
350 silly audit   xml2js: [ '0.4.19' ],
350 silly audit   '@mapbox/sphericalmercator': [ '1.1.0' ],
350 silly audit   '@mapbox/tiletype': [ '0.3.1' ],
350 silly audit   'd3-queue': [ '2.0.3' ],
350 silly audit   sqlite3: [ '4.2.0' ],
350 silly audit   ms: [ '2.0.0', '2.1.3' ],
350 silly audit   nan: [ '2.16.0' ],
350 silly audit   'node-pre-gyp': [ '0.11.0' ],
350 silly audit   'detect-libc': [ '1.0.3' ],
350 silly audit   needle: [ '2.9.1' ],
350 silly audit   nopt: [ '4.0.3' ],
350 silly audit   'npm-packlist': [ '1.4.8' ],
350 silly audit   npmlog: [ '4.1.2' ],
350 silly audit   rc: [ '1.2.8' ],
350 silly audit   rimraf: [ '2.7.1' ],
350 silly audit   semver: [ '5.7.1' ],
350 silly audit   tar: [ '4.4.19' ],
350 silly audit   'iconv-lite': [ '0.4.24' ],
350 silly audit   sax: [ '1.2.4' ],
350 silly audit   'safer-buffer': [ '2.1.2' ],
350 silly audit   abbrev: [ '1.1.1' ],
350 silly audit   osenv: [ '0.1.5' ],
350 silly audit   'ignore-walk': [ '3.0.4' ],
350 silly audit   'npm-bundled': [ '1.1.2' ],
350 silly audit   'npm-normalize-package-bin': [ '1.0.1' ],
350 silly audit   minimatch: [ '3.1.2' ],
350 silly audit   'brace-expansion': [ '1.1.11' ],
350 silly audit   'balanced-match': [ '1.0.2' ],
350 silly audit   'concat-map': [ '0.0.1' ],
350 silly audit   'are-we-there-yet': [ '1.1.7' ],
350 silly audit   'console-control-strings': [ '1.1.0' ],
350 silly audit   gauge: [ '2.7.4' ],
350 silly audit   'set-blocking': [ '2.0.0' ],
350 silly audit   delegates: [ '1.0.0' ],
350 silly audit   aproba: [ '1.2.0' ],
350 silly audit   'has-unicode': [ '2.0.1' ],
350 silly audit   'object-assign': [ '4.1.1' ],
350 silly audit   'signal-exit': [ '3.0.7' ],
350 silly audit   'string-width': [ '1.0.2' ],
350 silly audit   'strip-ansi': [ '3.0.1' ],
350 silly audit   'wide-align': [ '1.1.5' ],
350 silly audit   'os-homedir': [ '1.0.2' ],
350 silly audit   'os-tmpdir': [ '1.0.2' ],
350 silly audit   'deep-extend': [ '0.6.0' ],
350 silly audit   ini: [ '1.3.8' ],
350 silly audit   'strip-json-comments': [ '2.0.1' ],
350 silly audit   glob: [ '7.2.3' ],
350 silly audit   'fs.realpath': [ '1.0.0' ],
350 silly audit   inflight: [ '1.0.6' ],
350 silly audit   once: [ '1.4.0' ],
350 silly audit   'path-is-absolute': [ '1.0.1' ],
350 silly audit   wrappy: [ '1.0.2' ],
350 silly audit   'code-point-at': [ '1.1.0' ],
350 silly audit   'is-fullwidth-code-point': [ '1.0.0' ],
350 silly audit   'number-is-nan': [ '1.0.1' ],
350 silly audit   'ansi-regex': [ '2.1.1' ],
350 silly audit   chownr: [ '1.1.4' ],
350 silly audit   'fs-minipass': [ '1.2.7' ],
350 silly audit   minipass: [ '2.9.0' ],
350 silly audit   minizlib: [ '1.3.3' ],
350 silly audit   yallist: [ '3.1.1' ],
350 silly audit   xmlbuilder: [ '9.0.7' ],
350 silly audit   'process-nextick-args': [ '2.0.1' ]
350 silly audit }
351 http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 424ms
352 timing auditReport:getReport Completed in 433ms
353 silly audit report {}
354 timing auditReport:init Completed in 0ms
355 timing reify:audit Completed in 435ms
356 timing reifyNode:node_modules/process-nextick-args Completed in 498ms
357 timing reifyNode:node_modules/fs-minipass Completed in 498ms
358 timing reifyNode:node_modules/ansi-regex Completed in 497ms
359 timing reifyNode:node_modules/chownr Completed in 498ms
360 timing reifyNode:node_modules/is-fullwidth-code-point Completed in 498ms
361 timing reifyNode:node_modules/number-is-nan Completed in 498ms
362 timing reifyNode:node_modules/code-point-at Completed in 497ms
363 timing reifyNode:node_modules/wrappy Completed in 497ms
364 timing reifyNode:node_modules/path-is-absolute Completed in 498ms
365 timing reifyNode:node_modules/once Completed in 498ms
366 timing reifyNode:node_modules/inflight Completed in 498ms
367 timing reifyNode:node_modules/strip-json-comments Completed in 497ms
368 timing reifyNode:node_modules/ini Completed in 498ms
369 timing reifyNode:node_modules/os-tmpdir Completed in 498ms
370 timing reifyNode:node_modules/wide-align Completed in 498ms
371 timing reifyNode:node_modules/strip-ansi Completed in 498ms
372 timing reifyNode:node_modules/string-width Completed in 499ms
373 timing reifyNode:node_modules/object-assign Completed in 499ms
374 timing reifyNode:node_modules/has-unicode Completed in 500ms
375 timing reifyNode:node_modules/aproba Completed in 500ms
376 timing reifyNode:node_modules/os-homedir Completed in 502ms
377 timing reifyNode:node_modules/brace-expansion Completed in 501ms
378 timing reifyNode:node_modules/npm-bundled Completed in 501ms
379 timing reifyNode:node_modules/ignore-walk Completed in 501ms
380 timing reifyNode:node_modules/osenv Completed in 501ms
381 timing reifyNode:node_modules/abbrev Completed in 501ms
382 timing reifyNode:node_modules/npm-packlist Completed in 503ms
383 timing reifyNode:node_modules/ms Completed in 502ms
384 timing reifyNode:node_modules/needle/node_modules/ms Completed in 506ms
385 timing reifyNode:node_modules/minizlib Completed in 515ms
386 timing reifyNode:node_modules/minipass Completed in 515ms
387 timing reifyNode:node_modules/signal-exit Completed in 512ms
388 timing reifyNode:node_modules/set-blocking Completed in 512ms
389 timing reifyNode:node_modules/console-control-strings Completed in 512ms
390 timing reifyNode:node_modules/yallist Completed in 522ms
391 timing reifyNode:node_modules/fs.realpath Completed in 521ms
392 timing reifyNode:node_modules/balanced-match Completed in 519ms
393 timing reifyNode:node_modules/rimraf Completed in 518ms
394 timing reifyNode:node_modules/deep-extend Completed in 526ms
395 timing reifyNode:node_modules/minimatch Completed in 525ms
396 timing reifyNode:node_modules/npmlog Completed in 524ms
397 timing reifyNode:node_modules/are-we-there-yet/node_modules/string_decoder Completed in 529ms
398 timing reifyNode:node_modules/are-we-there-yet/node_modules/safe-buffer Completed in 529ms
399 timing reifyNode:node_modules/sax Completed in 530ms
400 timing reifyNode:node_modules/are-we-there-yet/node_modules/isarray Completed in 534ms
401 timing reifyNode:node_modules/delegates Completed in 536ms
402 timing reifyNode:node_modules/concat-map Completed in 536ms
403 timing reifyNode:node_modules/are-we-there-yet Completed in 541ms
404 timing reifyNode:node_modules/safer-buffer Completed in 539ms
405 timing reifyNode:node_modules/nopt Completed in 539ms
406 timing reifyNode:node_modules/detect-libc Completed in 539ms
407 timing reifyNode:node_modules/glob Completed in 548ms
408 timing reifyNode:node_modules/semver Completed in 543ms
409 timing reifyNode:node_modules/xml2js Completed in 549ms
410 timing reifyNode:node_modules/d3-queue Completed in 552ms
411 timing reifyNode:node_modules/@mapbox/mbtiles Completed in 552ms
412 timing reifyNode:node_modules/needle/node_modules/debug Completed in 555ms
413 timing reifyNode:node_modules/rc Completed in 562ms
414 timing reifyNode:node_modules/@mapbox/sphericalmercator Completed in 561ms
415 timing reifyNode:node_modules/npm-normalize-package-bin Completed in 576ms
416 timing reifyNode:node_modules/debug Completed in 575ms
417 timing reifyNode:node_modules/gauge Completed in 588ms
418 timing reifyNode:node_modules/xmlbuilder Completed in 615ms
419 timing reifyNode:node_modules/are-we-there-yet/node_modules/readable-stream Completed in 612ms
420 timing reifyNode:node_modules/tar Completed in 615ms
421 timing reifyNode:node_modules/iconv-lite Completed in 628ms
422 timing reifyNode:node_modules/baconjs Completed in 626ms
423 warn deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
424 timing reifyNode:node_modules/node-pre-gyp Completed in 631ms
425 timing reifyNode:node_modules/@mapbox/tiletype Completed in 633ms
426 timing reifyNode:node_modules/needle Completed in 661ms
427 timing reifyNode:node_modules/nan Completed in 675ms
428 timing reifyNode:node_modules/bluebird Completed in 674ms
429 timing reifyNode:node_modules/@signalk/charts-plugin Completed in 676ms
430 timing reifyNode:node_modules/sqlite3 Completed in 724ms
431 timing reifyNode:node_modules/lodash Completed in 953ms
432 timing reify:unpack Completed in 966ms
433 timing reify:unretire Completed in 0ms
434 timing build:queue Completed in 6ms
435 timing build:link:node_modules/@mapbox/sphericalmercator Completed in 4ms
436 timing build:link:node_modules/@mapbox/tiletype Completed in 3ms
437 timing build:link:node_modules/detect-libc Completed in 2ms
438 timing build:link:node_modules/needle Completed in 2ms
439 timing build:link:node_modules/node-pre-gyp Completed in 2ms
440 timing build:link:node_modules/nopt Completed in 2ms
441 timing build:link:node_modules/rc Completed in 2ms
442 timing build:link:node_modules/rimraf Completed in 3ms
443 timing build:link:node_modules/semver Completed in 2ms
444 timing build:link Completed in 5ms
445 info run sqlite3@4.2.0 install node_modules/sqlite3 node-pre-gyp install --fallback-to-build
446 info run sqlite3@4.2.0 install { code: 1, signal: null }
447 timing reify:rollback:createSparse Completed in 295ms
448 timing reify:rollback:retireShallow Completed in 0ms
449 timing command:install Completed in 11422ms
450 verbose stack Error: command failed
450 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:63:27)
450 verbose stack     at ChildProcess.emit (node:events:513:28)
450 verbose stack     at maybeClose (node:internal/child_process:1093:16)
450 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
451 verbose pkgid sqlite3@4.2.0
452 verbose cwd /home/raetara/.signalk
453 verbose Linux 5.15.0-48-generic
454 verbose node v16.17.1
455 verbose npm  v8.19.2
456 error code 1
457 error path /home/raetara/.signalk/node_modules/sqlite3
458 error command failed
459 error command sh -c -- node-pre-gyp install --fallback-to-build
460 error make: Entering directory '/home/raetara/.signalk/node_modules/sqlite3/build'
460 error   ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3310100/sqlite3.c
460 error make: Leaving directory '/home/raetara/.signalk/node_modules/sqlite3/build'
460 error Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64 --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
461 error node-pre-gyp info it worked if it ends with ok
461 error node-pre-gyp info using node-pre-gyp@0.11.0
461 error node-pre-gyp info using node@16.17.1 | linux | x64
461 error node-pre-gyp WARN Using needle for node-pre-gyp https download 
461 error node-pre-gyp info check checked for "/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" (not found)
461 error node-pre-gyp http GET https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz
461 error node-pre-gyp http 403 https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz
461 error node-pre-gyp WARN Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz 
461 error node-pre-gyp WARN Pre-built binaries not found for sqlite3@4.2.0 and node@16.17.1 (node-v93 ABI, glibc) (falling back to source compile with node-gyp) 
461 error node-pre-gyp http 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz 
461 error gyp info it worked if it ends with ok
461 error gyp info using node-gyp@9.1.0
461 error gyp info using node@16.17.1 | linux | x64
461 error gyp info ok 
461 error gyp info it worked if it ends with ok
461 error gyp info using node-gyp@9.1.0
461 error gyp info using node@16.17.1 | linux | x64
461 error gyp info find Python using Python version 3.10.6 found at "/usr/bin/python3"
461 error gyp info spawn /usr/bin/python3
461 error gyp info spawn args [
461 error gyp info spawn args   '/usr/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
461 error gyp info spawn args   'binding.gyp',
461 error gyp info spawn args   '-f',
461 error gyp info spawn args   'make',
461 error gyp info spawn args   '-I',
461 error gyp info spawn args   '/home/raetara/.signalk/node_modules/sqlite3/build/config.gypi',
461 error gyp info spawn args   '-I',
461 error gyp info spawn args   '/usr/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
461 error gyp info spawn args   '-I',
461 error gyp info spawn args   '/home/raetara/.cache/node-gyp/16.17.1/include/node/common.gypi',
461 error gyp info spawn args   '-Dlibrary=shared_library',
461 error gyp info spawn args   '-Dvisibility=default',
461 error gyp info spawn args   '-Dnode_root_dir=/home/raetara/.cache/node-gyp/16.17.1',
461 error gyp info spawn args   '-Dnode_gyp_dir=/usr/lib/node_modules/npm/node_modules/node-gyp',
461 error gyp info spawn args   '-Dnode_lib_file=/home/raetara/.cache/node-gyp/16.17.1/<(target_arch)/node.lib',
461 error gyp info spawn args   '-Dmodule_root_dir=/home/raetara/.signalk/node_modules/sqlite3',
461 error gyp info spawn args   '-Dnode_engine=v8',
461 error gyp info spawn args   '--depth=.',
461 error gyp info spawn args   '--no-parallel',
461 error gyp info spawn args   '--generator-output',
461 error gyp info spawn args   'build',
461 error gyp info spawn args   '-Goutput_dir=.'
461 error gyp info spawn args ]
461 error gyp info ok 
461 error gyp info it worked if it ends with ok
461 error gyp info using node-gyp@9.1.0
461 error gyp info using node@16.17.1 | linux | x64
461 error gyp info spawn make
461 error gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
461 error /bin/sh: 1: python: not found
461 error make: *** [deps/action_before_build.target.mk:13: Release/obj/gen/sqlite-autoconf-3310100/sqlite3.c] Error 127
461 error gyp ERR! build error 
461 error gyp ERR! stack Error: `make` failed with exit code: 2
461 error gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:201:23)
461 error gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
461 error gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
461 error gyp ERR! System Linux 5.15.0-48-generic
461 error gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v93"
461 error gyp ERR! cwd /home/raetara/.signalk/node_modules/sqlite3
461 error gyp ERR! node -v v16.17.1
461 error gyp ERR! node-gyp -v v9.1.0
461 error gyp ERR! not ok 
461 error node-pre-gyp ERR! build error 
461 error node-pre-gyp ERR! stack Error: Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/home/raetara/.signalk/node_modules/sqlite3/lib/binding/node-v93-linux-x64 --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
461 error node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/home/raetara/.signalk/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
461 error node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
461 error node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1093:16)
461 error node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
461 error node-pre-gyp ERR! System Linux 5.15.0-48-generic
461 error node-pre-gyp ERR! command "/usr/bin/node" "/home/raetara/.signalk/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
461 error node-pre-gyp ERR! cwd /home/raetara/.signalk/node_modules/sqlite3
461 error node-pre-gyp ERR! node -v v16.17.1
461 error node-pre-gyp ERR! node-pre-gyp -v v0.11.0
461 error node-pre-gyp ERR! not ok
462 verbose exit 1
463 timing npm Completed in 11702ms
464 verbose unfinished npm timer reify 1664334418018
465 verbose unfinished npm timer reify:build 1664334426547
466 verbose unfinished npm timer build 1664334426548
467 verbose unfinished npm timer build:deps 1664334426548
468 verbose unfinished npm timer build:run:install 1664334426560
469 verbose unfinished npm timer build:run:install:node_modules/sqlite3 1664334426560
470 verbose code 1
471 error A complete log of this run can be found in:
471 error     /home/raetara/.npm/_logs/2022-09-28T03_06_57_754Z-debug-0.log
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "email": "mikko.vesikkala@iki.fi"
   },
   "dependencies": {
-    "@mapbox/mbtiles": "0.10.0",
+    "@mapbox/mbtiles": "^0.12.1",
     "baconjs": "1.0.1",
     "bluebird": "3.5.1",
     "debug": "3.1.0",

--- a/src/plugin.js
+++ b/src/plugin.js
@@ -25,7 +25,7 @@ module.exports = function(app) {
       result[provider.identifier] = provider
       return result
     }, {})
-    debug(`Start charts plugin. Chart paths: ${chartPaths.join(', ')}, online charts: ${onlineProviders.length}`)
+    console.log(`Start charts plugin. Chart paths: ${chartPaths.join(', ')}, online charts: ${onlineProviders.length}`)
 
     const loadProviders = Promise.mapSeries(chartPaths, chartPath => Charts.findCharts(chartPath))
       .then(list => _.reduce(list, (result, charts) => _.merge({}, result, charts), {}))

